thanos/charts/values.yaml

```diff
@@ -36,36 +34,39 @@ thanos:
       routePrefix:
 
     additionalArgs: []
-      #- something.route-foo=/lol 
-
-    deploymentLabels: {} 
-
+      # - something.route-foo=/lol
+    deploymentLabels: {}
     containerLabels:
 
     serviceLabels:
 
     annotations:
 
-    stores: [] 
+    stores: []
 
   store:
     # adding additional arguments
     additionalArgs: []
 
-    deploymentLabels: {} 
+    deploymentLabels: {}
 
     containerLabels:
 
-    #TODO use
+    # TODO use
```

Comment:
do we need this comment?

Reply:
we leave it in for now, I remember I wanted to look at it.